### ISSUE TITLE
Wishlist - create md5 manifest from harvested hashes

### ISSUE THREAD

**Issue by kieranjol** [avarchival/AS11scripts#4]
If all hashes successfully validate after running as11fixity.py, it would be great if it would make an md5 manifest in the form of

`md5_checksum  relative/path/to/file.mxf`

as this will allow us to not only easily validate all the checksums in future, but if it's a sidecar, it can be used by moveit.py, skipping the need to generate the source manifest and speeding up that whole process.

What do ye think? @anjamahler @ecodonohoe

**Comment by ghost** [comment]
Yep, sounds like a logical addition for the whole as11 workflow.
